channel-manager-api.md

```diff
@@ -209,6 +201,157 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
 | `7` | Closed to departure |
 | `8` | Closed to stay |
 
+When all restrictions removed, state |`1`| is sent.
+
+
+```javascript
+       {
+           "ratePlanCode": "r",
+           "spaceTypeCode": "JST",
+           "state": [
+               1
+           ],
+           "minLos": null,
+           "maxLos": null,
+           "from": "2020-09-24",
+           "to": "2020-10-31"
+       }
+```
+
+New restrictions always override old restrictions. State |`1`| is not sent to remove old restrictions, if they were modified.
+
+State |`2`| is always sent in combination with state |`6`|, |`7`|, or |`8`|, or all together.
+
+```javascript
+        {
+            "ratePlanCode": "Ff",
+            "spaceTypeCode": "DEL",
+            "state": [
+                2,
+                8,
+                6,
+                7
+            ],
+            "minLos": null,
+            "maxLos": null,
+            "from": "2020-09-24",
+            "to": "2020-10-31"
+        }
+```
+Closed to Stay
+
+```javascript
+{
+            "ratePlanCode": "Ff",
+            "spaceTypeCode": "4BD",
+            "state": [
+                2,
+                8
+            ],
+            "minLos": null,
+            "maxLos": null,
+            "from": "2020-09-30",
+            "to": "2020-10-06"
+        }
+```
+Closed to Stay witn MinLos. State |`1`| is sent with specified minLos. If |`minLos`| is not met, then should be treated as closed.
+
+```javascript
+{
+    "ratePlanCode": "Ff",
+    "spaceTypeCode": "JST",
+    "state": [
+        1
+    ],
+    "minLos": 2,
+    "maxLos": 10,
+    "from": "2020-10-01",
+    "to": "2020-10-14"
+}
+
+Closed to Arrival
+
+```javascript
+            "ratePlanCode": "Ff",
+            "spaceTypeCode": "STA",
+            "state": [
+                2,
+                6
+            ],
+            "minLos": null,
+            "maxLos": null,
+            "from": "2020-09-30",
+            "to": "2020-10-06"
+        }
+```
+
+Closed to Arrival with MinLos. State |`1`| is sent with specified minLos. If |`minLos`| is not met, then should be treated as closed to arrival.
+
+```javascript
+{
+    "ratePlanCode": "Ff",
+    "spaceTypeCode": "JST",
+    "state": [
+        1
```

Comment:
I think by think they cannot know its close to arrival there is just open.
cc @ondrahermanek if we have closed to arrival with min or max los we sent open so they cannot know its closed to arrival

Reply:
@Ksenia-Galtsova can you make a YouTrack for this please? This should be better formatted in the API.